build/Targets/Packages.props

```diff
@@ -250,4 +251,28 @@
     <xunitrunnervisualstudioVersion>2.3.1</xunitrunnervisualstudioVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <RestoreSources>
+      $(RestoreSources);
+      https://dotnet.myget.org/F/dotnet-coreclr/api/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-corefxtestdata/api/v3/index.json;
+      https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
+      https://dotnet.myget.org/F/symreader/api/v3/index.json;
+      https://dotnet.myget.org/F/symreader-portable/api/v3/index.json;
+      https://dotnet.myget.org/F/symreader-converter/api/v3/index.json;
+      https://dotnet.myget.org/F/symreader-native/api/v3/index.json;
+      https://dotnet.myget.org/F/metadata-tools/api/v3/index.json;
+      https://dotnet.myget.org/F/interactive-window/api/v3/index.json;
+      https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json;
+      https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json;
+      https://dotnet.myget.org/F/roslyn-analyzers/api/v3/index.json;
+      https://dotnet.myget.org/F/roslyn/api/v3/index.json;
+      https://myget.org/F/vs-devcore/api/v3/index.json;
+      https://myget.org/F/vs-editor/api/v3/index.json;
+      https://vside.myget.org/F/vssdk/api/v3/index.json;
+      https://vside.myget.org/F/vs-impl/api/v3/index.json;
+    </RestoreSources>
+  </PropertyGroup>
```

Comment:
These values should be in a different file. Possibly Settings.props

Reply:
Why not put the sources in Versions.props here as well?

Reply:
Versions.props in Roslyn is different from Versions.props in other repos. In Roslyn it doesn't list the dependency versions. Roslyn's Versions.props corresponds more to toolset's [Version.props](https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/Version.props) and [Version.targets](https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/Version.targets)